Source/WTF/wtf/SystemTracing.h

```diff
@@ -52,6 +52,8 @@ enum TracePointCode {
     DumpJITMemoryStop,
     FromJSStart,
     FromJSStop,
```

Comment:
Pre-existing, but I noticed these don't have entries in the plist file. Should they, to make it easier for people using Ariadne to profile one off pages that use tracePointStart/tracePointEnd?

Reply:
I think those are exposed as "JSGlobalObject signpost".